haros/haros.py

```diff
@@ -673,9 +682,9 @@ def _save_results(self, node_cache):
     # dir/data/<name>
 
 class HarosExportRunner(HarosCommonExporter):
-    def __init__(self, haros_dir, data_dir, export_viz, project,
+    def __init__(self, haros_dir, config_dir, data_dir, export_viz, project,
```

Comment:
Same as above. Replace `config_dir` with `config_path`, since you are expecting a full path to the config file.